renovate.json

```diff
@@ -28,11 +28,15 @@
         "podinfo"
       ],
       "groupName": "podinfo"
+    },
+    {
+      "fileMatch": ["action.yaml"],
+      "groupName": "actions"
     }
   ],
   "regexManagers": [
     {
-      "fileMatch": ["uds-bundle.yaml", "action.yaml"],
+      "fileMatch": ["uds-bundle.yaml"],
```

Comment:
Might be missing something else here but by removing from this list it will no longer be picked up by renovate as part of this regex match. I think you might want to leave this, but also add the packageRule/group?

Reply:
I had thought that instruction would move them and I wouldn't want that there, but perhaps you're right and what I added was just grouping. I'll change that.

Reply:
Yeah so `packageRules` is used for grouping (and some other purposes, but we mainly use it for grouping). `regexManagers` is used to target specific strings to identify dependencies to update. So in this case we want to target updates to `action.yaml` files but now want them grouped in an `actions` group, so I think we'll want both.

Reply:
probably better now?

Reply:
wait nvm I overwrote my changes with a rebase, just a sec